asserter/errors.go

```diff
@@ -70,8 +70,11 @@ var (
 	ErrRelatedOperationIndexOutOfOrder = errors.New(
 		"related operation has index greater than operation",
 	)
-	ErrRelatedOperationIndexDuplicate     = errors.New("found duplicate related operation index")
-	ErrRelatedOperationMissing            = errors.New("related operations key is missing")
+	ErrRelatedOperationIndexDuplicate  = errors.New("found duplicate related operation index")
```

Comment:
nit: align all `=` in the same vertical line..

Reply:
formatting is correct. it's the github view messing around :)